run.py

```diff
@@ -0,0 +1,77 @@
+import autoUniCalendar
+import os
+import uuid
+import utils
+import re
+
+
+from flask import Flask, render_template, request, send_file
+from flask_talisman import Talisman
+app = Flask(__name__)
+Talisman(app, content_security_policy=None)
+
+defaultFilename = autoUniCalendar.filename
+debug = os.environ.get('FLASK_ENV') == 'development'
+
+@app.route('/', methods = ['GET'])
+def index():
+    return serve()
+
+@app.route('/', methods = ['POST'])
+def form_post():
+    if debug: print(f"[DEBUG] POST data received from React: {request.form}")
+
+    jsessionid = request.form.get('cookie1')
+    filename = defaultFilename
+    location = True
+    classType = True
+    extension = ".ics"
+
+    if debug:
+        print(f"[DEBUG] Calendar info: {jsessionid} → {filename}{extension}")
+        print(f"[DEBUG] Location parsing: {location}")
+        print(f"[DEBUG] Class type parsing: {classType}")
+        print(f"[DEBUG] iCalendar mode: {extension == '.ics'}")
+
+    if utils.verifyCookieExpiration(jsessionid):
```

Comment:
The check should be outside the main code, throwing an exception, instead of writing the code inside the if.

```suggestion
    if not utils.verifyCookieExpiration(jsessionid):
           raise Exception("The cookie is expired")
    # main code
```

Reply:
fair suggestion. I will resolve this later today.

Reply:
solved in [88fe237b4ae2619554d26a34e316bda7b8bf41c4.](https://github.com/miermontoto/epiCalendar/commit/88fe237b4ae2619554d26a34e316bda7b8bf41c4)